packages/dashboard/src/components/app-base.tsx

```diff
@@ -73,7 +65,6 @@ export function AppBase({ children }: React.PropsWithChildren<{}>): JSX.Element
   return (
     <ThemeProvider theme={theme}>
       <CssBaseline />
-      {settings.themeMode === ThemeMode.RmfDark && <GlobalStyles styles={rmfDarkLeaflet} />}
```

Comment:
This PR still keeps the themes now right? This should probably be reverted even if we are not exposing it in the ui.

Reply:
Good point, it'll be cleaned up when we clean up leaflet stuff next time, https://github.com/open-rmf/rmf-web/pull/910/commits/3b9f7279f9cbe0b08da4d93e354ed3eccb2f69a4